src/components/MenuLink.scss

```diff
@@ -0,0 +1,24 @@
+.link_text {
```

Comment:
I would keep class naming format consistent throughout the application. I would recommend using kebab-case, as suggested above, for all class names